src/osp/tasks/top_session.h

```diff
@@ -55,6 +61,17 @@
 #define OSP_AUX_DGDI_B(x) x
```

Comment:
Why is this "DGDI", and not "DCDI" ?

Reply:
Nevermind this, the review was hiding some important details so i misread what was going on.